### ISSUE TITLE
Relabel data

### ISSUE THREAD

**Issue by 123wk45678** [naver-ai/relabel_imagenet#1]
if i use our data,do not use ImageNet data,so,how do we get Relabel data?

**Comment by hellbell** [comment]
@123wk45678 
The code for computing the prediction map is not provided yet and we may update it later. 
Getting ReLabel data for your own data is not that hard. You can use a pre-trained strong classifier (e.g., EfficientNet-L2) and get the prediction map by removing the global average pooling layer as described in our paper's appendix.

**Comment by 123wk45678** [comment]
ok,thank you ,expect this to open sorce your code ,author

**Comment by 123wk45678** [comment]
ok,thank you ,expect this to open source your code ,author

**Comment by hammer422** [comment]
A bit confusing here.As described in paper,remove the global average pooling layer and add conv1x1 layer, do we need to fine-tuning this conv1x1 layer for getting label map?

**Comment by hellbell** [comment]
@changewOw 
Indeed, the `1x1conv` layer is exactly the same as the final `fully-connected` layer. 
The size of the fully-connected layer's weight matrix `[d x C]` (d=channel size, C=number of classes), 
and we reshape it to `[1 x 1 x d x C]` to be the kernel matrix of `1x1conv`. 
We present the detailed descriptions in Appendix A of [our paper](https://arxiv.org/pdf/2101.05022.pdf).

**Comment by hammer422** [comment]
Got it! Thank you very much

**Comment by hellbell** [comment]
@changewOw You're welcome :)
